pkg/utils/utils.go

```diff
@@ -181,6 +209,27 @@ func SyncNodeState(newState *sriovnetworkv1.SriovNetworkNodeState, pfsToConfig m
 				continue
 			}
 
+			// load the PF info
+			pfStatus, exist, err := LoadPfsStatus(ifaceStatus.PciAddress, false)
+			if err != nil {
+				glog.Errorf("SyncNodeState(): failed to load info about PF status for pci address %s: %v", ifaceStatus.PciAddress, err)
+				return err
+			}
+
+			if !exist {
+				glog.Infof("SyncNodeState(): PF name %s with pci address %s as vfs configured but they didn't created by the sriov operator skipping the device reset",
```

Comment:
nit:

`SyncNodeState(): PF name %s with pci address %s has VFs configured but they weren't created by the sriov operator. Skipping the device reset`